.travis.yml

```diff
@@ -23,20 +23,20 @@ env:
     - ROS_DISTRO=hydro  USE_DEB=false      NOT_TEST_INSTALL=true
     - ROS_DISTRO=hydro  USE_DEB=true       NOT_TEST_INSTALL=true INSTALL_SRC="http://github.com/jsk-ros-pkg/jsk_pr2eus" TEST_PKGS="pr2eus"
     - ROS_DISTRO=hydro  USE_DEB=true       NOT_TEST_INSTALL=true BEFORE_SCRIPT="git clone -b hydro-devel http://github.com/ros/geometry; git clone -b hydro-devel http://github.com/ros/geometry-experimental; touch jsk_roseus/roseus_tutorials/CATKIN_IGNORE; touch geometry-experimental/test_tf2/CATKIN_IGNORE" # need to ignore roseus_tutorials which depends on image_view2 and that install deb version of tf/tf2
-    - ROS_DISTRO=hydro  USE_DEB=source
-    - ROS_DISTRO=indigo USE_DEB=true
-    - ROS_DISTRO=indigo USE_DEB=true       ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - ROS_DISTRO=indigo USE_DEB=false      NOT_TEST_INSTALL=true
-    - ROS_DISTRO=indigo USE_DEB=source
-    - ROS_DISTRO=jade USE_DEB=true       TEST_PKGS="roseus roseus_smach" NOT_TEST_INSTALL=true
-    - ROS_DISTRO=jade USE_DEB=true       ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu TEST_PKGS="roseus roseus_smach" NOT_TEST_INSTALL=true
-    - ROS_DISTRO=jade USE_DEB=false      NOT_TEST_INSTALL=true TEST_PKGS="roseus roseus_smach"
-    - ROS_DISTRO=jade USE_DEB=source     TEST_PKGS="roseus roseus_smach" NOT_TEST_INSTALL=true
+    - ROS_DISTRO=hydro  USE_DEB=source     BEFORE_SCRIPT="sudo apt-get install -y xvfb"
+    - ROS_DISTRO=indigo USE_DEB=true       BEFORE_SCRIPT="sudo apt-get install -y xvfb"
+    - ROS_DISTRO=indigo USE_DEB=true       ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu BEFORE_SCRIPT="sudo apt-get install -y xvfb"
+    - ROS_DISTRO=indigo USE_DEB=false      NOT_TEST_INSTALL=true BEFORE_SCRIPT="sudo apt-get install -y xvfb"
+    - ROS_DISTRO=indigo USE_DEB=source   BEFORE_SCRIPT="sudo apt-get install -y xvfb"
```

Comment:
i think you cam remove all BEFORE_SCIRPT because of `sudo apt-get install
xvfb` in `before_script:` section of travis.yml

--
◉ Kei Okada


On Thu, Nov 12, 2015 at 12:44 PM, Furushchev <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/jsk-ros-pkg/jsk_roseus/pull/393#discussion_r44619038>:
>
> > @@ -23,20 +23,20 @@ env:
> >      - ROS_DISTRO=hydro  USE_DEB=false      NOT_TEST_INSTALL=true
> >      - ROS_DISTRO=hydro  USE_DEB=true       NOT_TEST_INSTALL=true INSTALL_SRC="http://github.com/jsk-ros-pkg/jsk_pr2eus" TEST_PKGS="pr2eus"
> >      - ROS_DISTRO=hydro  USE_DEB=true       NOT_TEST_INSTALL=true BEFORE_SCRIPT="git clone -b hydro-devel http://github.com/ros/geometry; git clone -b hydro-devel http://github.com/ros/geometry-experimental; touch jsk_roseus/roseus_tutorials/CATKIN_IGNORE; touch geometry-experimental/test_tf2/CATKIN_IGNORE" # need to ignore roseus_tutorials which depends on image_view2 and that install deb version of tf/tf2
> > -    - ROS_DISTRO=hydro  USE_DEB=source
> > -    - ROS_DISTRO=indigo USE_DEB=true
> > -    - ROS_DISTRO=indigo USE_DEB=true       ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
> > -    - ROS_DISTRO=indigo USE_DEB=false      NOT_TEST_INSTALL=true
> > -    - ROS_DISTRO=indigo USE_DEB=source
> > -    - ROS_DISTRO=jade USE_DEB=true       TEST_PKGS="roseus roseus_smach" NOT_TEST_INSTALL=true
> > -    - ROS_DISTRO=jade USE_DEB=true       ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu TEST_PKGS="roseus roseus_smach" NOT_TEST_INSTALL=true
> > -    - ROS_DISTRO=jade USE_DEB=false      NOT_TEST_INSTALL=true TEST_PKGS="roseus roseus_smach"
> > -    - ROS_DISTRO=jade USE_DEB=source     TEST_PKGS="roseus roseus_smach" NOT_TEST_INSTALL=true
> > +    - ROS_DISTRO=hydro  USE_DEB=source     BEFORE_SCRIPT="sudo apt-get install -y xvfb"
> > +    - ROS_DISTRO=indigo USE_DEB=true       BEFORE_SCRIPT="sudo apt-get install -y xvfb"
> > +    - ROS_DISTRO=indigo USE_DEB=true       ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu BEFORE_SCRIPT="sudo apt-get install -y xvfb"
> > +    - ROS_DISTRO=indigo USE_DEB=false      NOT_TEST_INSTALL=true BEFORE_SCRIPT="sudo apt-get install -y xvfb"
> > +    - ROS_DISTRO=indigo USE_DEB=source   BEFORE_SCRIPT="sudo apt-get install -y xvfb"
>
> removed BEFORE_SCRIPT from ROS_DISTRO=hydro USE_DEB=source
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jsk-ros-pkg/jsk_roseus/pull/393/files#r44619038>.
>